liquality/wallet
# [TEST]-Added BTC send to invalid address and check Review button has been di…

### Improvements 💶

- Included page object pattern in the Puppeter framework
- Each page class should have page actions

### TESTS 💻
### ### TEST 1 🎯
- Added Element ID for SEND view
- BTC send to an invalid address and check the error message
- Check Review button has been disabled
### ### TEST 2 🎯
- Try to SEND with exceed amount and check the error message
- Send Review button has been Disabled
### docs 📚
- Updated Readme docs with tests
- Fix some existing docs
### dependency upgrade 🛠

- Upgrade eslint to latest version
